src/views/ipv/page/confirm-address.njk

```diff
@@ -48,7 +48,7 @@
             },
             items: [
                     {
-                    value: "addressCurrent",
+                    value: "address-current",
```

Comment:
In terms of consistency why are we adding a hyphen here but removing one from `driving-licence` below?

Reply:
This is because these reflect the format of the corresponding journey event